frontend/src/Components/Chat.js

```diff
@@ -1,18 +1,18 @@
-import React, { Component } from 'react';
-// import axios from 'axios';
-
-class Chat extends Component {
-  constructor() {
-    super();
-    this.state = {
-      message: {
-        time: Date.now(),
-        message: '',
-      },
-    };
-  }
-  render() {
-    return <div className='chat-container'>Chat</div>;
-  }
-}
+import React, { Component } from 'react';
+// import axios from 'axios';
+
+class Chat extends Component {
+  constructor() {
+    super();
+    this.state = {
+      message: {
+        time: Date.now(),
+        message: '',
+      },
+    };
+  }
+  render() {
+    return <div className='chat-container'>Chat</div>;
+  }
+}
```

Comment:
We may have an issue with different code formatters going on if GH thinks the same code is actually different code.  Let's make sure we're all using the same thing (or nothing at all)